src/api/event.ts

```diff
@@ -55,8 +55,10 @@ export const deleteEvent = (eid: string) => Delete<{}>('event/' + eid);
 export const getUpcomingEvents = (): Promise<Event[]> =>
   get<Event[]>('event/upcoming');
 
-export const getPastEvents = (): Promise<Event[]> =>
-  get<Event[]>('event/past-events');
+export const getPastEvents = (skip: number, limit: number): Promise<Event[]> =>
```

Comment:
It's generally fine to send `limit` as a dynamic value here to the endpoint. However, because you're only using a constant value of `10` at the moment, could it be better/easier to simply set it to `10` statically in the endpoint? Both solutions are okay in this case